py2hwsw/lib/hardware/basic_tests/iob_aoi/iob_aoi.py

```diff
@@ -120,27 +120,21 @@
             "width": "W",
         },
     ],
-    "buses": [
+    "wires": [
         {
-            "name": "and_ab_out",
+            "name": "aab",
             "descr": "and ab output",
-            "wires": [
-                {"name": "aab", "width": "W"},
-            ],
+            "width": "W",
         },
         {
-            "name": "and_cd_out",
+            "name": "cad",
             "descr": "and cd output",
-            "wires": [
-                {"name": "cad", "width": "W"},
-            ],
+            "width": "W",
         },
         {
-            "name": "or_out",
+            "name": "oab",
```

Comment:
```suggestion
            "name": "aob",
```
[nitpick] This signal should be renamed for consistency with the previous ones.
"aob" meaning "a or b".

Similar to previous signal:
"cad" meaning "c and d"